src/backend/gl/src/device.rs

```diff
@@ -233,10 +233,11 @@ impl Device {
                 gl.use_program(Some(program));
             }
             for (name, &(register, slot)) in name_binding_map.iter() {
+                log::trace!("Get binding {:?} from program {:?}", name, program);
```

Comment:
nit: the backend currently imports the macros from `log`, so it can be just `trace!` here.
At the end of the day, I'd prefer all the code to use the style you used here with fully-qualified log, but it's not very important to fix.

Reply:
Fixed in separate commit. I'm also prefer `log::` style.